test/suites/basic.sh

```diff
@@ -103,8 +103,11 @@ test_basic_usage() {
   # Delete the bar container we've used for several tests
   lxc delete bar
 
-  # lxc delete should also delete all snapshots of bar
-  [ ! -d "${LXD_DIR}/snapshots/bar" ]
+  # FIXME: make this backend agnostic
+  if [ "${LXD_BACKEND}" = "dir" ]; then
+    # lxc delete should also delete all snapshots of bar
+    [ ! -d "${LXD_DIR}/snapshots/bar" ]
```

Comment:
I'm surprised that this code doesn't work with other backends, I thought that both btrfs and lvm were setting up things in there and I know for a fact that zfs does.